src/patternfly/components/Wizard/wizard.scss

```
@@ -48,6 +48,14 @@
   --#{$wizard}__nav-link--m-disabled--Color: var(--pf-t--global--text--color--disabled);
   --#{$wizard}__nav-link--m-disabled--BackgroundColor: transparent;
 
+  // Nav link status icon
+  --#{$wizard}__nav-link-icon--Color: var(--pf-t--global--icon--color--regular);
+  --#{$wizard}__nav-link--m-danger--nav-link-icon--Color: var(--pf-t--global--icon--color--status--danger--default);
```

Technically I think you'd use `--m-danger__nav-link-icon` (since it's a nested element) but since the var starts at `__nav-link` I think you'd also be alright just calling this `__icon` if you wanted to shorten the var.

```suggestion
  --#{$wizard}__nav-link--m-danger__icon--Color: var(--pf-t--global--icon--color--status--danger--default);
```